BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -1773,9 +1784,9 @@ private void UpdateEngineLists(bool forceUpdate = false)
                     reverseEngines.Add(engine);
                     reverseThrust += engine.MaxThrustOutputVac(true);
                 }
-                else 
+                else if (EngineRCSRotation || EngineRCSTranslation)
```

Comment:
Are engines that can't be shut down included in these lists? They should probably be excluded.